src/commands/InstallCommand.ts

```diff
@@ -139,4 +139,8 @@ export interface InstallCommandArgs {
      * The list of packages that should be installed
      */
     packages?: string[];
+    /**
+     * Dependencies installation location
```

Comment:
Can you elaborate a bit here for future readers? Since we usually load rootDir from package.json, it would be useful to explain that's where the value is found initially, and this property will override that.